charts/airbyte/templates/env-configmap.yaml

```diff
@@ -72,4 +72,6 @@ data:
   WORKER_STATE_STORAGE_TYPE:  {{ .Values.global.state.storage.type | quote }}
   SHOULD_RUN_NOTIFY_WORKFLOWS: "false"
   MAX_NOTIFY_WORKERS: {{ .Values.worker.maxNotifyWorkers | default "5" | quote }}
+  STRICT_COMPARISON_NORMALIZATION_TAG: ""
```

Comment:
Is that value is expected to be always empty?
If not, then can you please add an Values in both airbyte and airbyte-worker charts?

Reply:
This is new, I tried to look for documentation but I had no success, I think it could be established by now